state-chain/pallets/cf-ingress-egress/src/lib.rs

```diff
@@ -822,7 +822,7 @@ impl<T: Config<I>, I: 'static> Pallet<T, I> {
 											.unwrap_or(false)
 									},
 								),
-								FetchOrTransfer::Transfer { .. } => true,
+								FetchOrTransfer::Transfer { amount, .. } => !amount.is_zero(),
```

Comment:
Lower down in this method, we subtract some fees from this amount. This is where the check needs to happen (line 866).

Reply:
Oh...